client/components/mma/accountoverview/manageProducts/ManageProductV2.tsx

```diff
@@ -0,0 +1,384 @@
+			</h2>
+
+			<section
+				css={css`
+					border-bottom: 1px solid ${palette.neutral[86]};
+					padding-bottom: ${space[5]}px;
+				`}
+			>
+				<Stack space={5}>
+					<div
+						css={css`
+							border-top: 1px solid ${palette.neutral[86]};
+							padding-bottom: ${space[1]}px;
+						`}
+					></div>
+					<ul
+						css={[
+							iconListCss,
+							listWithDividersCss,
+							whatHappensNextIconCss,
+						]}
+					>
+						<li>
+							<SvgClock size="medium" />
+							<span>
+								<>
+									<strong
+										css={css`
+											padding-bottom: ${space[1]}px;
+										`}
+									>
+										Next {monthlyOrAnnual}
+										{''} payment
+									</strong>
+									<br /> {nextPaymentDetails?.paymentValue}
+								</>
+							</span>
+						</li>
+						<li>
+							<SvgCalendar size="medium" />
+							<span>
+								<>
+									<strong
+										css={css`
+											padding-bottom: ${space[1]}px;
+										`}
+									>
+										Next payment date
+									</strong>{' '}
+									<br />
+									{nextPaymentDetails?.nextPaymentDateValue}
+								</>
+							</span>
+						</li>
+						<li>
+							<SvgCreditCard size="medium" />
+							<span data-qm-masking="blocklist">
+								<strong
+									css={css`
+										padding-bottom: ${space[1]}px;
+									`}
+								>
+									Payment Method
+								</strong>
+								<br />
+								<PaymentDetails
+									subscription={productDetail.subscription}
+								/>
+							</span>
+						</li>
+					</ul>
+				</Stack>
+			</section>
+
+			<section
+				css={css`
+					margin-top: ${space[4]}px;
+				`}
+			>
+				<div css={stackedButtonLayoutCss}>
+					{productDetail.isPaidTier &&
+						productDetail.subscription.safeToUpdatePaymentMethod &&
+						!productDetail.subscription.payPalEmail && (
+							<LinkButton
+								href={`/payment/${specificProductType.urlPart}`}
+								cssOverrides={buttonCentredCss}
+							>
+								Update payment method
+							</LinkButton>
+						)}
+
+					<div
+						css={css`
+							margin-left: ${space[5]}px;
+						`}
+					>
+						{!hasCancellationPending && (
+							<CancellationCTA
+								productDetail={productDetail}
+								friendlyName={groupedProductType.friendlyName()}
+								specificProductType={specificProductType}
+							/>
+						)}
+					</div>
+				</div>
+			</section>
+		</>
+	);
+};
+
+interface CancellationCTAProps {
+	productDetail: ProductDetail;
+	friendlyName: string;
+	specificProductType: ProductType;
+}
+
+const CancellationCTA = (props: CancellationCTAProps) => {
+	const shouldContactUsToCancel =
+		!props.productDetail.selfServiceCancellation.isAllowed ||
+		!props.specificProductType.cancellation;
+	return (
+		<>
+			{shouldContactUsToCancel && (
+				<div
+					css={css`
+						${textSans.medium()};
+						color: ${palette.neutral[46]};
+						margin-top: 16px;
+						justify-content: center;
+					`}
+				>
+					Would you like to cancel your {props.friendlyName}?
```

Comment:
in the figma it looks like it just says "Cancel subscription"

Reply:
This is temporary until the self cancellation flag is set to true...we could change it at the moment but I would prefer to keep it as is so we remember to do it!